dashboard/test/fixtures/gatekeeper_invalid.yaml

```diff
@@ -0,0 +1,35 @@
+# This yaml file describes the configuration for
+# gatekeeper.  Here is how a typical feature would
+# be defined:
+#
+# <feature name>:
+# - rule: <descripion of what rule does>
```

Comment:
Could we call this "comment" or "description"?   "rule" sounds like the system actually interprets this string which is not the case.